src/src/WebServer/WebServer.cpp

```diff
@@ -221,6 +221,9 @@ void WebServerInit()
   webserver_init = true;
 
   // Prepare webserver pages
+
+  // FIXME TD-er: The added String() wrapper is needed for the latest ESP32 core lib.
+  // See: https://github.com/espressif/arduino-esp32/issues/4374
   #ifdef WEBSERVER_ROOT
   web_server.on("/",               handle_root);
```

Comment:
Shouldn't that `"/"` also have the `F()` macro, like the other strings?

Reply:
Probably, but I've had some issues not so long ago with the web_server.on calls with FlashStringHelper on ESP32.
So perhaps I missed this one?

Reply:
> perhaps I missed this one?

That's the most likely cause 😉 
Not a biggie, of course, just something I spotted.